lib/screen/note_screen.dart

```diff
@@ -298,28 +309,25 @@ class _NoteScreen extends State<NoteScreen> with CommandHandler {
   /// Callback on a single note clicked
   void _onNoteTap(Note note) async {
     final command =
-    await Navigator.pushNamed(context, '/note_editor', arguments: {'note': note});
+    await Navigator.pushNamed(context, '/note_editor', arguments: {'note': note, 'bookId': _book.id});
     processNoteCommand(_scaffoldKey.currentState, command);
   }
 
   /// Create notes query
   Stream<List<Note>> _createNoteStream(
       BuildContext context, NoteFilter filter) {
-    final user = Provider.of<CurrentUser>(context)?.data;
+    final user = FirebaseAuth.instance.currentUser;
     final sinceSignUp = DateTime.now().millisecondsSinceEpoch -
         (user?.metadata?.creationTime?.millisecondsSinceEpoch ?? 0);
     final useIndexes = sinceSignUp >=
         _10_min_millis; // since creating indexes takes time, avoid using composite index until later
     final collection = bookNotesCollection(_book.id, user?.uid);
-    final query = filter.noteState == NoteState.unspecified
-        ? collection
+    final query =collection
         .where('state',
```

Comment:
Here by not sending the index, I was able to fix the bug where after creating the notes, you can't see theses notes. It seems like we need to create index before using indexes